src/ExpressionEvaluator/Core/Test/ExpressionCompiler/ExpressionCompilerTestHelpers.cs

```diff
@@ -128,8 +128,10 @@ internal static class ExpressionCompilerTestHelpers
             ExpressionCompiler.CreateContextDelegate createContext,
             DkmUtilities.GetMetadataBytesPtrFunction getMetaDataBytesPtr,
             out string errorMessage,
+            out DiagnosticBag diagnosticBag,
```

Comment:
Does this method still need the 'errorMessage' parameter if it now exposes all diagnostics?